packages/manager/tools/sao-ovh-manager-app/lib/update-pkg.js

```diff
@@ -10,25 +10,26 @@ module.exports = ({ name }) => ({
   license: 'BSD-3-Clause',
   author: 'OVH SAS',
   scripts: {
-    build: 'yarn build:eu && yarn build:ca && yarn build:us',
-    'build:ca': "webpack --env.production --env.region='CA' && mv dist dist-CA",
-    'build:eu': 'webpack --env.production && mv dist dist-EU',
-    'build:us': "webpack --env.production --env.region='US' && mv dist dist-US",
+    build: 'webpack --env.production',
     dev: 'webpack-dev-server',
     'dev:watch': 'yarn run dev',
     start: `lerna exec --stream --scope='@ovh-ux/manager-${name}-app' --include-dependencies -- npm run build --if-present`,
     'start:dev': `lerna exec --stream --scope='@ovh-ux/manager-${name}-app' --include-dependencies -- npm run dev --if-present`,
     'start:watch': `lerna exec --stream --parallel --scope='@ovh-ux/manager-${name}-app' --include-dependencies -- npm run dev:watch --if-present`,
   },
   dependencies: {
-    '@ovh-ux/manager-config': '^0.4.0',
-    '@ovh-ux/manager-core': '^9.0.0',
+    '@ovh-ux/manager-config': '^1.0.0',
+    '@ovh-ux/manager-core': '^10.0.0',
```

Comment:
**through:** `sao-ovh-manager-app` and `sao-ovh-manager-module` are linked in some way, so maybe we can also update the range here (https://github.com/ovh/manager/blob/feat/app-template/packages/manager/tools/sao-ovh-manager-module/lib/update-pkg.js#L15) to `^10.0.0`

Is that make sense for you? And maybe later we can consider finding a way to automatically retrieve the latest version when scaffolding.

Reply:
Yes it makes sense, I'll update